Library/Formula/osquery.rb

```diff
@@ -16,12 +16,11 @@ class Osquery < Formula
   depends_on :macos => :mavericks
 
   depends_on "cmake" => :build
-
-  depends_on "boost"
-  depends_on "gflags"
-  depends_on "openssl"
-  depends_on "rocksdb"
-  depends_on "thrift"
+  depends_on "boost" => :build
```

Comment:
yup, we statically link in all of these deps, so it was a bit of an oversight on our part that we were depending on them. let's see if it blends!